packages/extension/manifest.json

```diff
@@ -5,5 +5,6 @@
   "background": {
     "service_worker": "background.js"
   },
-  "permissions": ["contextMenus", "scripting", "activeTab"]
+  "permissions": ["contextMenus", "scripting", "activeTab", "tabs"],
```

Comment:
Correct me if I'm wrong but I don't think we should need the 'tabs' permission?

Reply:
ahh nope, had it for zendesk stuff.